volttron/platform/main.py

```diff
@@ -873,10 +886,16 @@ def rmq_router(stop):
         # Launch additional services and wait for them to start before
         # auto-starting agents
         services = [
-            ControlService(opts.aip, address=address, identity='control',
-                           tracker=tracker, heartbeat_autostart=True,
-                           enable_store=False, enable_channel=True,
-                           message_bus=opts.message_bus),
+            ControlService(
+                opts.aip, address=address, identity='control',
+                tracker=tracker, heartbeat_autostart=True,
+                enable_store=False, enable_channel=True,
+                message_bus=opts.message_bus,
+                agent_monitor_frequency=opts.agent_monitor_frequency),
+
+            CompatPubSub(address=address, identity='pubsub.compat',
```

Comment:
CompatPubSub is deprecated as of VOLTTRON 6.0. Please remove it.

Reply:
Done